bin/strata-cli/src/seed.rs

```diff
@@ -196,14 +196,13 @@ pub fn load_or_create(
 
         let mut password = Password::read(true).map_err(OneOf::new)?;
         let entropy = password.entropy();
-        let _ = term.write_line(format!("Password strength (Overall strength score from 0-4, where anything below 3 is too weak): {}", entropy.score()).as_str());
         if entropy.score() <= Score::Two {
             let _ = term.write_line(
-                entropy
-                    .feedback()
-                    .expect("No feedback")
-                    .to_string()
-                    .as_str(),
+                format!(
+                    "Password is weak. {}",
+                    entropy.feedback().expect("No feedback")
+                )
```

Comment:
I would do `.unwrap_or("no feedback")` or whatever is needed to make the lifetimes work.  Also a colon instead of a period there for aesthetics.